### PR TITLE
[BLOCKED] Update invoke task and add "requires" entry

### PR DESCRIPTION
**Pull request by CasperWA**

### CODE REVIEW

**Comment by CasperWA** [comment]
@csadorf Is it fine to add metadata.requires entries like this? See the change [here](https://github.com/aiidalab/aiidalab-optimade/pull/28/files#diff-12e598daecade60b04e493f45c134fec9287ac7471114d95d1995b4fb651a6c8R9-R16).

File: metadata.json [hunk 1]
```diff
@@ -9,6 +9,9 @@
     "requires": {
         ">=2.0.1": [
```
Comment (csadorf) [inline metadata.json:10]:
```suggestion
        ">=2.0.1,<2.0.3": [
```
In this way it is non-ambiguous.

Comment (CasperWA) [reply metadata.json:10]: Okay. And this addition is required and cannot be deduced automatically by the system from the current setup?
I'm automating this part, hence I'd like to have as few edits for it as possible 😅

Comment (csadorf) [reply metadata.json:10]: No, while the implied prevalence may seem obvious in simple cases like this, there is no way to automatically deduce which constraint should take prevalence in general. However, exactly for this reason, we'll revise the app store, to reduce/eliminate the need to manually specify constraints like this. Will discuss at today's AiiDAlab meeting.

Comment (CasperWA) [reply metadata.json:10]: Then I'll wait slightly with updating this until after the meeting.

**Comment by csadorf** [comment]
Please do not merge this until after today's meeting.

**Comment by CasperWA** [comment]
> Please do not merge this until after today's meeting.

Not merging this at all until the new system has been put in place.

**Comment by yakutovicha** [comment]
I am going to archive this repo, so the PR is closed.

**Pull request closed without merging**